Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -88,41 +91,44 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
 x180_len = 40
 x180_sigma = x180_len / 5
 x180_amp = 0.35
-x180_wf, x180_der_wf = np.array(drag_gaussian_pulse_waveforms(x180_amp, x180_len, x180_sigma, alpha=0, delta=1))
+x180_wf, x180_der_wf = np.array(drag_gaussian_pulse_waveforms(x180_amp, x180_len, x180_sigma, alpha=drag_coef, delta=1))
 # No DRAG when alpha=0, it's just a gaussian.
 
 x90_len = x180_len
 x90_sigma = x90_len / 5
 x90_amp = x180_amp / 2
-x90_wf, x90_der_wf = np.array(drag_gaussian_pulse_waveforms(x90_amp, x90_len, x90_sigma, alpha=0, delta=1))
+x90_wf, x90_der_wf = np.array(drag_gaussian_pulse_waveforms(x90_amp, x90_len, x90_sigma, alpha=drag_coef, delta=1))
 # No DRAG when alpha=0, it's just a gaussian.
 
 minus_x90_len = x180_len
 minus_x90_sigma = minus_x90_len / 5
 minus_x90_amp = -x90_amp
 minus_x90_wf, minus_x90_der_wf = np.array(
-    drag_gaussian_pulse_waveforms(minus_x90_amp, minus_x90_len, minus_x90_sigma, alpha=0, delta=1)
+    drag_gaussian_pulse_waveforms(minus_x90_amp, minus_x90_len, minus_x90_sigma, alpha=drag_coef, delta=1)
 )
 # No DRAG when alpha=0, it's just a gaussian.
 
 y180_len = x180_len
 y180_sigma = y180_len / 5
 y180_amp = x180_amp
-y180_wf, y180_der_wf = np.array(drag_gaussian_pulse_waveforms(y180_amp, y180_len, y180_sigma, alpha=0, delta=1))
+y180_wf, y180_der_wf = np.array(drag_gaussian_pulse_waveforms(y180_amp, y180_len, y180_sigma, alpha=drag_coef, delta=1))
+y180_der_wf = (-1) * y180_der_wf
```

Comment:
no sign is wrong in the py qua tools. The sign is related to writing `y180` instead of doing frame rotation.

I'm assuming the frame matrix is ((C, -S), (S,C)). For frame = 0 -> ((1,0),(0,1)). For frame = pi/2 -> ((0, -1),(1, 0)).

For frame =0 then I can load "I" = "I_wf" and "Q" = "Q_wf" for rotations along -x

When I want to write rotations along -y, I need to load "I" = -"Q_wf" and "Q" = "I_wf".

The way it was written before was that "I" = "Q_wf" and "Q" = "I_wf" which comes from frame matrix ((0, 1), (1,0)) that based on  ((C, -S), (S,C)) it would not be for any angle.

What do you think?

Reply:
Yes, I see.
It is a bit confusing, maybe simply adding a #I waveform comment after it works be enough?
Or alternatively we can do:
`y180_I_wf = -y180_der_wf`
And same for Q.
Or keep as is...
Opinions?

Reply:
I like your idea of `y180_I_wf` and I do think it is confusing but I went for it anyways :) -- will implement